cloudwatch_alb_target_group/main.tf

```diff
@@ -38,22 +40,52 @@ resource "aws_cloudwatch_metric_alarm" "healthy_hosts_low_too_long" {
 
 resource "aws_cloudwatch_metric_alarm" "healthy_hosts_seriously_low" {
   alarm_name = "${var.app_environment}:alb:public:${var.targets_name} Healthy Hosts: Severe Deficiency"
-  alarm_description = "Significantly less than the desired number of healthy ${var.targets_name} hosts behind the ALB"
-  namespace = "AWS/ApplicationELB"
-  dimensions = {
-    "LoadBalancer" = var.alb_dimension_id
-    "TargetGroup" = var.target_group_dimension_id
-  }
-  metric_name = "HealthyHostCount"
+  alarm_description = "Significantly less than the desired proportion of healthy ${var.targets_name} hosts behind the ALB"
   comparison_operator = "LessThanThreshold"
-  threshold = var.server_min_instances - 1
-  unit = "Count"
-  period = "60"
-  statistic = "Average"
+  threshold = var.server_min_healthy_proportion
   evaluation_periods = "3"
   alarm_actions = [var.sns_monitoring_topic_arn]
   ok_actions = [var.sns_monitoring_topic_arn]
   insufficient_data_actions = []
+
+  metric_query {
+    id          = "p1"
```

Comment:
Can these metrics have more descriptive names? h1, h2 and p1 are hard to understand at a glance.